src/lancie-login/lancie-logout-button.html

```diff
@@ -0,0 +1,58 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/lancie-ajax/lancie-ajax.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<dom-module id="lancie-logout-button">
+  <template>
+    <style>
+      :host {
+        display: block;
+        margin: 4px;
+      }
+
+      paper-icon-button {
+        color: var(--secondary-color);
+        height: 40px;
+        width: 40px;
+      }
+    </style>
+
+    <lancie-ajax id="ajaxLogout" refurl="logout" method="POST" on-lancie-ajax="onLogoutResponse"></lancie-ajax>
+    <div on-tap="tryLogout">
+      <paper-icon-button icon="lancie:exit-to-app"></paper-icon-button>
+      <span hidden="[[!labeled]]">Sign out</span>
+    </div>
+
+  </template>
+  <script>
+    class LancieLogoutButton extends Polymer.Element {
+      static get is() {
+        return 'lancie-logout-button';
+      }
+
+      static get properties() {
+        return {
+          labeled: {
+            type: Boolean,
+            value: false,
+          }
+        }
```

Comment:
Missing semicolumn here: causing the build to fail
```suggestion
        };
```